signal_scanner_bot/messages.py

```diff
@@ -51,6 +52,50 @@ def _condense_command(message: str) -> str:
     return NON_ALPHA_NUMERIC.sub("", message).upper()
 
 
+def _strip_tweet_hashtags(status_text: str) -> str:
```

Comment:
Great idea moving these out to separate functions! Though these three should probably be in the `twitter.py` module, do you mind moving them there?

Reply:
I'm a little conflicted now that I'm thinking about this. The code to reformat Signal messages for Twitter content is homed in the twitter module, so by that logic we should home the code for reformatting Twitter content for Signal messages in the signal module, but maybe it would be better to put the functions in the modules for the data they're acting on, e.g. putting the code to break up the Signal messages into serialized tweets should be in the signal module, which eventually passes the strings down into the function from the twitter module to send the tweets. Or we could break the string manipulation stuff into their own module? I dunno, I'm super conflicted now.

Maybe we should do this PR and then discuss it and make an issue to rationalize it in a subsequent PR?

Reply:
Ya know, this all definitely needs another look. I'm with you - let's get this in then take a second stab at it in another PR :smile: